### DISCUSSION TITLE
Can’t cancel searches using the app bar

### DISCUSSION THREAD

**Discussion by tombulled** [community/community#13460] [Mobile]
### Steps to reproduce
1. Visit the 'Home' tab (by clicking the home icon in the bottom app bar)
2. Perform a search (focus the search bar, enter a query, hit enter)
3. Attempt to re-visit the 'Home' tab (by clicking the home icon in the bottom app bar)

### Expected behaviour
In iOS it is expected that this will cancel the current search and return you to the home page. Such behaviour is currently implement for other views, however this doesn't appear to be the case for searches.

### Reasoning
Mobile users, myself included, are often lazy and don't want to reach up to the top-right of the screen to cancel the search using the 'Cancel' button. If this feature was implemented, searches could easily be cancelled using the bottom app bar, which is much more reachable

**Comment by brianlovin** [comment]
Agreed! I've opened an issue for us to fix this behavior, thanks @tombulled 😊
